crates/rolldown_testing/src/integration_test.rs

```diff
@@ -660,6 +661,8 @@ impl IntegrationTest {
         node_command.arg("--eval");
         node_command.arg("\"\"");
       });
+      // workaround for https://github.com/nodejs/node/issues/59374
+      node_command.arg("--input-type=module");
```

Comment:
This was the reason the test I added in #5618 wasn't failing
https://github.com/nodejs/node/issues/59374